### ISSUE TITLE
Search box missing form label - icons.brand.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa801#9738]
# Motivation

Manual testing has revealed the search box on icons.brand.uiowa.edu is missing a form label. This is an accessibility compliance issue.

# Additional Info

Not seeing in Siteimprove scan or browser plugin.

Is being reported by WAVE and the field label checker.

<img width="1750" height="1013" alt="Image" src="https://github.com/user-attachments/assets/e626a01d-1f3e-4d9b-878b-bd129d7cf649" />
